scripts/python/benchmarking/capture_rss_and_cpu.py

```diff
@@ -32,6 +32,18 @@ def __init__(self, pid: int, interval: float = 0.1):
         self.interval = interval
         self.output_file = f'process_{pid}_metrics.txt'
 
+        self.cpu_count_logical = cpu_count()  # logical cores (including hyperthreading)
+        # in cloud, this is known as vCPUs
```

Comment:
/nit format

Reply:
I tried indenting it to the right (to match the line above) but the linter complained about that and made me move it left. I might move both of them on top or below to make it look nicer.